packages/core/useVirtualList/demo.vue

```diff
@@ -1,43 +1,64 @@
 <script setup lang="ts">
-import { ref, Ref } from 'vue-demi'
+import { ref, Ref, computed } from 'vue-demi'
 import { useVirtualList } from '.'
 
-const index: Ref = ref(0)
+const index: Ref = ref()
+const search = ref('')
+
+const allItems = Array.from(Array(99999).keys())
+  .map(i => ({
+    height: i % 2 === 0 ? 42 : 84,
+    size: i % 2 === 0 ? 'small' : 'large',
+  }))
+
+const filteredItems = computed(() => {
+  return allItems.filter(i => i.size.startsWith(search.value.toLowerCase()))
+})
+
 const { list, containerProps, wrapperProps, scrollTo } = useVirtualList(
-  Array.from(Array(99999).keys()),
+  filteredItems,
   {
-    itemHeight: i => (i % 2 === 0 ? 42 + 8 : 84 + 8),
```

Comment:
Previously this was index-based, this does not work with the reactive list -- the heights are calculated incorrectly.